tests/test_0019-use-json-library.cpp

```diff
@@ -52,6 +52,15 @@ int main(int, char**) {
   slice.append(ak::SliceRange(ak::Slice::none(), ak::Slice::none(), 2));
   slice.append(ak::SliceRange(1, ak::Slice::none(), ak::Slice::none()));
 
+  auto cuda_arr =  array->to_gpu(kernel::Lib::cuda_kernels);
+  auto arr = cuda_arr->num(0, 1);
```

Comment:
Even in tests, we should not use integers for enum values. C++ doesn't enforce it, but enums should be set by the enum identifier, so that it can be reordered at any time in the future.

This practice is also good for when you need to search for all instances of an enum. `0` isn't very searchable.

Reply:
I am a bit confused here, are you talking about line `56`?(`auto arr = cuda_arr->num(0, 1);`). The `ListOffsetArrayOf::num` has two parameters, `axis` and `depth`. I don't think I am using integers for `enum` anywhere. And I should remove this from this file, since this file tests something else. I have added a similar test in `test_0293-listarray-num-kernel.cpp`.

Reply:
I misinterpreted `num(0, 1)`, thinking that the first argument is the `cpu-kernels` enum—my mistake! I think this is okay, then.